index.js

```diff
@@ -283,13 +284,15 @@ module.exports = class Hyperdrive extends EventEmitter {
 
   // atm always recursive, but we should add some depth thing to it
   list (folder, { recursive = true } = {}) {
+    folder = folder || '/'
```

Comment:
needs `if (typeof folder === 'object') return this.list(undefined, folder)` also then